pkg/controller/volume/persistentvolume/provision_test.go

```diff
@@ -102,6 +128,14 @@ var storageClasses = []*storage.StorageClass{
 		VolumeBindingMode: &modeImmediate,
 	},
 }
+var nodes = []*v1.Node{
+	{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   "node1",
+			Labels: map[string]string{"key1": "value1"},
+		},
+	},
+}
```

Comment:
it's only be used in one place, make it local in the function where it is used.

Reply:
fixed